pkg/wallet/wallet.go

```diff
@@ -31,10 +31,15 @@ const (
 	PBKDF2NbRound             = 600_000
 	FileModeUserReadWriteOnly = 0o600
 	Base58Version             = 0x00
+	AddressVersion            = 0x00
+	PubKeyVersion             = 0x00
+	SignatureVersion          = 0x00
+	PrivKeyVersion            = 0x00
 	UserAddressPrefix         = "AU"
 	PublicKeyPrefix           = "P"
 	PrivateKeyPrefix          = "S"
 	MaxNicknameLength         = 32
+	AccountVersion            = 1
```

Comment:
Other versions are in hexa, why not this one ?

Reply:
this is the version of the wallet file format standard, not the version of a blockchain element (address, keys, signature)

Reply:
see related PR: https://github.com/massalabs/massa-standards/pull/101/files